### ISSUE TITLE
using team pipelines

### ISSUE THREAD

**Issue by adeslatt** [lifebit-ai/splicing-pipelines-nf#1]
Using team pipelines - it isn't clear what the parameters are when you import them.   It would be helpful to see what parameters are expected and/or required.

**Comment by PhilPalmer** [comment]
Yeah, I agree, this would be nice to have. Diogo has worked on a feature to solve this problem actually but it's still in development unfortunately
